gui/src/types.ts

```diff
@@ -8,7 +8,7 @@ export const networkSchema = z.object({
         name: z.string().min(1),
         dev: z.boolean().default(false),
         http_url: z.string().min(1),
-        ws_url: z.string().optional(),
+        ws_url: z.string().nullable().optional(),
```

Comment:
@naps62 have a look at this one. I had some issues because even though the field was optional, the value 'null' was not acceptable, only undefined.

Reply:
nice. I don't remember if I even thought about that before